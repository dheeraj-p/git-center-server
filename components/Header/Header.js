import HeaderStyles from './Header.style';

export default function Header() {
  return (
    <nav className="navbar header">
      <h3>
        <small className="text-light">GitCenter</small>
      </h3>
      <style jsx>{HeaderStyles}</style>
    </nav>
  );
}
