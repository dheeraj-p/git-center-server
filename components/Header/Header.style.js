import css from 'styled-jsx/css';
import colors from '../../styles/colors';

const HeaderStyles = css`
  .header {
    background-color: ${colors.colorBackgroundDark};
  }
`;

export default HeaderStyles;
