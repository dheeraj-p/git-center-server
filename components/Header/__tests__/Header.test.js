import { shallow } from 'enzyme';
import Header from '../Header';

describe('Header', () => {
  it('should render properly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
