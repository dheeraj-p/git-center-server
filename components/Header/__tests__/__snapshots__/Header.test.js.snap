// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render properly 1`] = `
"<nav className=\\"jsx-1483795907 navbar header\\">
  <h3 className=\\"jsx-1483795907\\">
    <small className=\\"jsx-1483795907 text-light\\">
      GitCenter
    </small>
  </h3>
  <JSXStyle id=\\"1483795907\\">
    .
    h
    e
    a
    d
    e
    r
    .
    j
    s
    x
    -
    1
    4
    8
    3
    7
    9
    5
    9
    0
    7
    {
    b
    a
    c
    k
    g
    r
    o
    u
    n
    d
    -
    c
    o
    l
    o
    r
    :
    #
    1
    b
    2
    6
    2
    c
    ;
    }
  </JSXStyle>
</nav>"
`;
